model/job.go

```diff
@@ -71,7 +71,7 @@ type InputParameter struct {
 	Value string `json:"value" bson:"value"`
 
 	// Raw value of the input parameter
-	Raw interface{} `json:"-" bson:"raw"`
+	Raw interface{} `json:"raw,omitempty" bson:"raw"`
```

Comment:
You are right, historically we supported strings only, and then we introduced `Raw` when we added JSON support. Maybe it's time to refactor it. In this specific case, I needed Raw to be encoded in the JSON we publish to nats, otherwise is lost. I would prefer not touching this part in this PR...

Reply:
Agreed, let's change it separately from this PR.